docs/configuration.rst

```diff
@@ -167,3 +167,66 @@ not be compatible with non-standard deployments, and should only be used when no
 invocations would result in a large number of queries or when latency is particularly important.
 
 Defaults to ``False``.
+
+GUARDIAN_USER_OBJ_PERMS_USE
```

Comment:
Why not `GUARDIAN_USER_OBJ_PERMS_MODEL`?

Reply:
Good idea.  _MODEL would be clearer than _USE.
Updated names in commit 15eb65d95fa50c23761d0343812b598553b1c0ee